SampleProjects/DoSomething/.arduino-ci.yaml

```diff
@@ -11,3 +11,5 @@ unittest:
     - uno
     - due
     - leonardo
+  compilers:
```

Comment:
If this line is necessary to make things work, please open an issue for that... the `compilers` setting should be automatically inherited from https://github.com/ianfixes/arduino_ci/blob/master/misc/default.yml#L83-L84

Reply:
Ok. That is what I thought originally until I spotted TestSomething had it explicitly defined so I assumed it must have been a misunderstanding on my part. Note that this behaviour is the same on Windows and Linux. I'll raise an issue.